.ci/pipeline_definitions

```diff
@@ -12,6 +12,11 @@ gardener-extension-provider-alicloud:
             image: 'eu.gcr.io/gardener-project/gardener/extensions/provider-alicloud'
             dockerfile: 'Dockerfile'
             target: gardener-extension-provider-alicloud
+          gardener-extension-provider-alicloud:
```

Comment:
This should be `ardener-extension-validator-alicloud`

Reply:
good catch! Thanks a lot. I will fix it immediately.